code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -510,6 +512,29 @@ static HookFunction hookFunction([]()
 			}
 		});
 
+		fx::ScriptEngine::RegisterNativeHandler("OVERRIDE_PLAYER_TALKING", [](fx::ScriptContext& context)
```

Comment:
Since it's setting a persistent state, `SET_PLAYER_OVERRIDE_TALKING` (or `SET_PLAYER_OVERRIDE_TALK_STATE`, or maybe `SET_PLAYER_TALKING_STATE`, or similar) seems more correct a name.

I'd stick with `SET_PLAYER_TALKING_STATE`, unless R* used `OVERRIDE_` prefixes before.

Reply:
R* uses `OVERRIDE_` prefixes sometimes. So, I will use `SET_PLAYER_OVERRIDE_TALKING`, as you said.